client/src/game/ui/PlayStack.ts

```diff
@@ -44,7 +44,7 @@ export default class PlayStack extends Konva.Group {
         opacity,
         easing: Konva.Easings.EaseOut,
         onFinish: () => {
-          if (!layoutChild || !card || !card.parent) {
+          if (layoutChild === undefined || card === undefined || card.parent === undefined) {
```

Comment:
null check here

Reply:
nope, can't be null. you gotta check the types

Reply:
card.parent can't be null?
then why do you check if it is null in other places

Reply:
Because Konva, as you know, is very consistent. Parent can be null in some types, and can't in others.

Reply:
should we just check for parent === null everywhere? it makes the code more consistent, and could make the code more resilient if we ever upgrade konva

Reply:
That's the thing, it doesn't really, because TS will catch if you forget a null check.